packages/database/src/TupaiaDatabase.js

```diff
@@ -181,7 +183,7 @@ export class TupaiaDatabase {
 
   wrapInTransaction(wrappedFunction) {
     return this.connection.transaction(transaction =>
-      wrappedFunction(new TupaiaDatabase(transaction)),
+      wrappedFunction(new TupaiaDatabase(transaction, this.changeChannel)),
```

Comment:
@IgorNadj had to update the code a little to get the unit tests to work. They were hanging because now we are instantiating a `changeChannel` connection within a transaction, which was never being cleaned up. I switched it to pass through the parent connection's `changeChannel` though there may possibly still be issues with this.
1. It's possible for the `changeChannel` to still be `null` when we create the transaction, in which case the transaction will still create it's own if it needs one. This seems pretty unlikely though since we create the `changeChannel` most of the time...
2. The `changeChannel` and the transaction have separate database connections. This is actually always the case generally it seems. However, this does mean that notifications will be sent before the transaction completes... Honestly it seems a bit odd to me that the two have separate connections, but I feel it's too much of a refactor to change it right now...